lib/chef/resource/rekey.rb

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright:: Copyright (c) 2014-2019, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Chef
+  class Resource
+    class Rekey < Chef::Resource
+      provides :rekey
+      resource_name :rekey
```

Comment:
seems like this would make more sense if we named it client_rekey

Reply:
yeah that makes sense, will do.